var/spack/repos/builtin/packages/py-torchgeo/package.py

```diff
@@ -49,13 +51,14 @@ class PyTorchgeo(PythonPackage):
     depends_on("py-kornia@0.6.4:0.6.9", when="@0.3", type=("build", "run"))
     depends_on("py-kornia@0.5.11:0.6.9", when="@0.2", type=("build", "run"))
     depends_on("py-kornia@0.5.4:0.6.9", when="@0.1", type=("build", "run"))
+    depends_on("py-lightning@1.8:2", when="@0.4.1:", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("py-lightning@1.8:1", when="@0.4.1:", type=("build", "run"))
```
2 works, but we messed up the setup.cfg so we should match it until 0.4.2.